build-farm/platform-specific-configurations/mac.sh

```diff
@@ -51,6 +54,13 @@ else
     # JDK17 requires metal (included in full xcode) as does JDK11 on aarch64
     # JDK11 on x64 is matched for consistency
     XCODE_SWITCH_PATH="/Applications/Xcode.app"
+    # JDK11 and 17 on Mac (x86 and aarch) has excessive warnings.
+    # This is due to a harfbuzz fix which is pending backport.
+    # Suppressing the warnings for now to aid triage.
+    if [[ "$JAVA_FEATURE_VERSION" -le 17 ]]; then
```

Comment:
I thought this was only a problem for versions 11 and 17? This will presumably make the change for 8 as well so I would suggest explicitly checking for 11 or 17.

Reply:
Hi Stewart. This will not make the change for 8 as a result of the "if" in line 37:

```
if [ "${JAVA_TO_BUILD}" == "${JDK8_VERSION}" ]
...
else
...
Our code.
```

Reply:
Hmmm ok ... This is getting a bit complex to read but I'll approve.

Reply:
Thanks